docs/.vuepress/config.js

```diff
@@ -0,0 +1,146 @@
+  [
+    // Google Tag Manager, an extra element within the `ssr.html` file.
+    ['script', {}, `
+      (function(w,d,s,l,i){w[l]=w[l]||[];w[l].push({'gtm.start':
+        new Date().getTime(),event:'gtm.js'});var f=d.getElementsByTagName(s)[0],
+        j=d.createElement(s),dl=l!='dataLayer'?'&l='+l:'';j.async=true;j.src=
+        '//www.googletagmanager.com/gtm.js?id='+i+dl;f.parentNode.insertBefore(j,f);
+      })(window,document,'script','dataLayer','GTM-55L5D3');
+    `],
+  ]
+  : [];
+
+module.exports = {
+  patterns: ['**/*.md', '!README.md', '!README-EDITING.md', '!README-DEPLOYMENT.md'], // to enable vue pages add: '**/*.vue'.
+  description: 'Handsontable',
+  base: '/docs/',
+  head: [
+    ['link', { rel: 'icon', href: 'https://handsontable.com/static/images/template/ModCommon/favicon-32x32.png' }],
+    ['meta', { name: 'viewport', content: 'width=device-width, initial-scale=1' }],
+    // Cookiebot - cookie consent popup
+    ['script', {
+      id: 'Cookiebot',
+      src: 'https://consent.cookiebot.com/uc.js',
+      'data-cbid': 'ef171f1d-a288-433f-b680-3cdbdebd5646'
+    }],
+    ...environmentHead
+  ],
+  markdown: {
+    toc: {
+      includeLevel: [2, 3],
+      containerHeaderHtml: '<div class="toc-container-header">Table of contents</div>'
+    },
+  },
+  plugins: [
+    'tabs',
+    ['sitemap', {
+      hostname: 'https://handsontable.com',
+    }],
+    ['@vuepress/active-header-links', {
+      sidebarLinkSelector: '.table-of-contents a',
+      headerAnchorSelector: '.header-anchor'
+    }],
+    ['container', examples],
+    ['container', sourceCodeLink],
+    {
+      extendMarkdown(md) {
+        const render = function(tokens, options, env) {
+          let i; let type;
+          let result = '';
+          const rules = this.rules;
+
+          for (i = 0; i < tokens.length; i++) { // overwritten here
+            type = tokens[i].type;
+
+            if (type === 'inline') {
+              result += this.renderInline(tokens[i].children, options, env);
+            } else if (typeof rules[type] !== 'undefined') {
+              result += rules[tokens[i].type](tokens, i, options, env, this);
+            } else {
+              result += this.renderToken(tokens, i, options, env);
+            }
+          }
+
+          return result;
+        };
+
+        // overwrite markdown `render` function to allow extending tokens array (remove caching before loop).
+        md.renderer.render = (tokens, options, env) => render.call(md.renderer, tokens, options, env);
+      },
+      chainMarkdown(config) {
+        // inject custom markdown highlight with our snippet runner
+        config
+          .options
+          .highlight(highlight)
+          .end();
+      },
+      chainWebpack: (config) => {
+        config.module
+          .rule('md')
+          .test(/\.md$/)
+          .use(path.resolve(__dirname, 'docs-links'))
+          .loader(path.resolve(__dirname, 'docs-links'))
```

Comment:
AFAIK, `use` set a context, `loader` set a loader for this context. `loader()` should not work (or does not exist) without `use` used before. 

I may check once again it if you want.

Reply:
Ok

Reply:
![image](https://user-images.githubusercontent.com/2893099/123641207-4dc25080-d822-11eb-9104-8085a53a5e9a.png)

Reply:
Both are needed because that is the structure of webpack config. You can't have empty `use` and `loader` at the upper lvl.

```js
{
  use: {
    loader: path.resovle(...)
  }
}
```

But from the object structure I would expect it to have `use` empty: `use().loader(path.resolve(__dirname, 'docs-links'))` because path is set in nested `loader`

Reply:
It doesn't hold back the initial release. Let's do it at your earliest convenience @Siemienik

Reply:
Reported https://github.com/handsontable/handsontable/issues/8428.